build/Linux+BSD/portable/copy-libs

```diff
@@ -259,9 +260,9 @@ building_on_CentOS_6() {
   copyLib libjpeg.so.62
   copyLib libssl.so.10
   copyLib libtiff.so.3
-  copyLib libicudata.so.42 # moved from cross-platform
-  copyLib libicui18n.so.42 # moved from cross-platform
-  copyLib libicuuc.so.42 # moved from cross-platform
+  copyLib libicudata.so.56 # moved from cross-platform
+  copyLib libicui18n.so.56 # moved from cross-platform
+  copyLib libicuuc.so.56 # moved from cross-platform
```

Comment:
I had to update this to .56 or else reported this missing check-depends.  This .56 is provided by the qt5 zipfile, while .42 is provided by CentOS 6 repo.